pkg/core/structure.go

```diff
@@ -1,6 +1,6 @@
-// Package contains all core interfaces and basic implementations.
 package core
 
+// Structure is the representation of a structure.
```

Comment:
// Structure is the representation of the component tree structure.

Reply:
done